### DISCUSSION TITLE
Stating the support or lack of support for readthedocs builds?

### DISCUSSION THREAD

**Discussion by ssbarnea** [jdoiro3/mkdocs-multirepo-plugin#72] [General]
We do know that RTD offers support for multi-repo projects, via sub-projects feature. The big question is that if this project is compatible with RTD?

Mainly this translates to two questions:

- will subprocess still be able to use RTD mkdocs to build their standalone sites?
- will we be able to use a central repo to build the combined site?

RTD is a very important player in documentation building as is one of the very few that can easily build a new website on each pending change, so you can evaluate how a change impacts the documentation website.

BTW, Thanks for starting this plugin!

**Comment by oleg-nenashev** [comment]
Hi @ssbarnea, it has been a while since Jenkins times. I had a few experiments, and indeed it seems to work okay with RTD. My main concern is about rate limits, etc.
